app/services/base_response.py

```diff
@@ -0,0 +1,25 @@
+from flask import jsonify
+
+from app.controllers.base import BaseController
+
+
+class BaseResponse:
+    def __init__(self, controller: BaseController):
+        self.controller = controller
+
+    def receive_response(self):
```

Comment:
We could change the name of this method since its functionality is to send a request
```suggestion
    def send_request(self):
```